libvast/src/system/version_command.cpp

```diff
@@ -40,6 +40,7 @@ record combine(const record& lhs, const record& rhs) {
 record retrieve_versions() {
   record result;
   result["VAST"] = version::version;
+  result["VAST Build Type"] = version::build_type;
```

Comment:
Should we add the sanitizer configuration in here as well? We could have a `sanitizers` key.

Reply:
That's already in this PR: It renders as `"VAST Build Type": "Debug with Assertions, Address Sanitizer"`.

Reply:
Uh, that's not nice to parse. We're already using JSON, so while we're changing this, why not make it structured?

Reply:
The build type is not even supposed to be parsed; it's entirely customizable. There is no standardization around it—it's just an arbitrary string.

Reply:
And that's okay, I think. But let's not creep in more than needed into this opaque value. Why not make it easy for users to check whether a given sanitizer is on? Here's an example of what I mean:

```json
"Build Settings": {
  "Assertions": true,
  "Address Sanitizier": true,
  "UB Sanitizier": false,
}
```

Reply:
The main reason I did not do it this way is because that's (1) more changes to `config.hpp.in` and (2) easily 3x the amount of CMake I need to write. But I can make that change. :-)